packages/utils/src/assetData/baseAssets.ts

```diff
@@ -121,6 +121,21 @@ export const thorchain: Readonly<Asset> = Object.freeze({
   relatedAssetKey: null,
 })
 
+export const mayachain: Readonly<Asset> = Object.freeze({
+  assetId: caip.mayachainAssetId,
+  chainId: caip.mayachainChainId,
+  name: 'MAYAChain',
+  networkName: 'MAYAChain',
+  symbol: 'CACAO',
+  precision: 10,
+  color: '#63FDD9',
+  icon: 'https://gitlab.com/mayachain/frontend/mayachain-explorer-v2/-/raw/main/assets/images/cacao.png',
+  explorer: 'https://mayascan.org',
+  explorerAddressLink: 'https://mayascan.org/address/',
+  explorerTxLink: 'https://mayascan.org/tx/',
```

Comment:
the explorers for mayachain aren't great, as in they only really display swap txs, but mayascan at least shows _something_ for send/receive txs though it is essentially useless.